src/main/resources/schemas/opa_press_list.json

```diff
@@ -0,0 +1,600 @@
+                                              "description": "Object representing an informant",
+                                              "title": "Informant",
+                                              "type": "object",
+                                              "properties": {
+                                                "prosecutionAuthorityRef": {
+                                                  "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/hearing/case/informant/prosecutionAuthorityRef",
+                                                  "title": "Prosecution Authority Ref",
+                                                  "type": "string",
+                                                  "default": "",
+                                                  "examples": [
+                                                    ""
+                                                  ],
+                                                  "pattern": "^.*$"
+                                                }
+                                              }
+                                            }
+                                          }
+                                        }
+                                      },
+                                      "party": {
+                                        "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/hearing/party",
+                                        "title": "Party",
+                                        "description": "Parties involved in the case (defence, prosecution)",
+                                        "type": "array",
+                                        "items": {
+                                          "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/hearing/items",
+                                          "title": "Party",
+                                          "description": "A party",
+                                          "type": "object",
+                                          "properties": {
+                                            "partyRole": {
+                                              "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/hearing/items/partyRole",
+                                              "description": "Role of the party, selectable list of roles, i.e. prosecution, solicitor defence etc",
+                                              "title": "Party Role",
+                                              "type": "string",
+                                              "examples": [
+                                                "ACCUSED", "PROSECUTOR"
+                                              ],
+                                              "enum": ["DEFENDANT", "PROSECUTING_AUTHORITY"],
```

Comment:
Only minor, however theres a mismatch between examples + enum which could be confusing

Reply:
Have updated this